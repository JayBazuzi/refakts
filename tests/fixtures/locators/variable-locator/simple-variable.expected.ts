/**
 * Expected output: JSON describing variable declaration and all usages
 * Declaration: line 7, column 8 (const count = 42)
 * Usages: line 8, column 15 (console.log(count)) and line 9, column 9 (return count + 1)
 */

function example() {
  const count = 42;
  console.log(count);
  return count + 1;
}