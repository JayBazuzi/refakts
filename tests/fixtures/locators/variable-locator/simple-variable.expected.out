[simple-variable.received.ts 7:9-7:19] count = 42
[simple-variable.received.ts 8:15-8:20] count
[simple-variable.received.ts 9:10-9:15] count